### PR TITLE
Update Antonio_Ramos_Documentation

### PR DESCRIPTION
**Pull request by aramosri**

### CODE REVIEW

File: Antonio_Ramos_Documentation [hunk 1]
```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
...
+- got the following error back "RuntimeError: b'no arguments in initialization list'"
+
+- my guess is this is related to the problems in the python installation because it is the same issue I was facing when I was having problems with the python installation last month
+
+8/16/2020
+- finally emailed ICT about python installation
+
+- investigating the block data information now to understand where it came from and how it was structured when it was made
+
+- unsure about what program was used to merge the two files that are refereced in Will's "What I Did" document
```
Comment (kbuzard) [inline Antonio_Ramos_Documentation:460]: It's possible that he did a join in ArcGIS -- if you can't find a program that does it, then assume this is the case. He started out doing a lot of operations in ArcGIS and was converting it all over to doing it programatically, but I can't guarantee he finished the conversion

File: Antonio_Ramos_Documentation [hunk 2]
```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
...
+
+- will begin determining which order to install packages so that conflicts do not arise
+
+8/11/2020
+- removed the programTrial3.6 environment and created new environment where I install packages and programs in the following order 
+  1. Spyder 4.0.1. (Installed through the anaconda navigator) 
+  2. arcpy (anaconda command prompt command "conda install -c esri arcpy")
+  3. plotnine (anaconda command prompt "conda install -c conda-forge plotnine") 
+  4. geopandas (anaconda command prmopt "conda install-c conda-forge geopandas") 
+- all moduels and packed installed successfully
```
Comment (kbuzard) [inline Antonio_Ramos_Documentation:403]: Are there some typos here? It isn't clear what this means.

File: Antonio_Ramos_Documentation [hunk 3]
```diff
@@ -381,3 +381,80 @@ Tab2 = Tab2.astype(convert_dict)
 
 - Problems left to resolve - 1. no cluster file is being generated even after 499 simulations 2. the find and replace done on the excel file needs to be automated in python 3. the code that produces the labs data file is not producing files which project correctly 
 
+8/8/2020
+
+- installying arcpy to python envinronment "programTrial" using following command in anaconda command prompt "conda install -c esri arcpy"
```
Comment (kbuzard) [inline Antonio_Ramos_Documentation:386]: Please add some context here for why you're doing this, why you need arcpy, what program you're working on updating

File: Antonio_Ramos_Documentation (hunk 1 above)
Comment (aramosri) [reply Antonio_Ramos_Documentation:460]: Ok I will see if I can find a way to programmatize this join.

File: Antonio_Ramos_Documentation (hunk 2 above)
Comment (aramosri) [reply Antonio_Ramos_Documentation:403]: This section is just the order in which we have to install packages in the python environment to be able to install arcpy. If you do not go in this order when installing packages in a new python environment, then you will have conflicting dependencies between the packages.

File: Antonio_Ramos_Documentation (hunk 3 above)
Comment (aramosri) [reply Antonio_Ramos_Documentation:386]: Will do. Arcpy is a package giving us the recalculate command.

File: Antonio_Ramos_Documentation (hunk 2 above)
Comment (kbuzard) [reply Antonio_Ramos_Documentation:403]: Looking at this again, I'm thinking that "packed" should be "packages"?

**Comment by aramosri** [comment]
Oh wow. I am so sorry even after reading it before commenting I didn’t notice that typo. Yes, “packed” is supposed to be “packages”. That’s awfully embarrassing on my end. My apologies.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: Antonio_Ramos_Documentation (hunk 3 above)
Comment (kbuzard) [reply Antonio_Ramos_Documentation:386]: Now I feel like I should have been able to figure it out. But I was trying to figure out what "packed" meant in this context and got blinded by it

**Comment by kbuzard** [comment]
These clarifications are really helpful--thanks!

**Pull request merged**